armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -283,14 +283,6 @@ def axiallyExpandAssembly(self):
                     c.ztop = c.zbottom + c.height
                     # redistribute block boundaries if on the target component
                     if self.expansionData.isTargetComponent(c):
-                        if not b.p.axialExpTargetComponent:
-                            runLog.debug(
-                                f"      Component {c} is target component (inferred)"
-                            )
-                        else:
-                            runLog.debug(
-                                f"      Component {c} is target component (blueprints defined or loaded from DB)"
-                            )
```

Comment:
@keckler I know you're a fan of these, and they are coming back with [this feature branch](https://github.com/albeanth/armi/tree/rmComponentDeterminesBlockHeight) so they won't be gone long.